README.md

```diff
@@ -30,16 +30,53 @@ month: 2
 year: 2023
 ---
 
-your markdown content
+# This is a heading style one
+
+**This is a featured Paragraph it cannot contain bold text but can contain [links](https://www.unitary.fund).**
```

Comment:
What flavor of markdown is this? Is a featured paragraph the same as a bolded paragraph?
```suggestion
**This is a featured paragraph it cannot contain bold text but can contain [links](https://www.unitary.fund).**
```